applications/GeoMechanicsApplication/GeoMechanicsApplication.py

```diff
@@ -0,0 +1,7 @@
+# Application dependent names and paths
+from KratosMultiphysics import _ImportApplication
+from KratosGeoMechanicsApplication import *
```

Comment:
here you should also import the Apps this app depends on (aka StructuralMechanics)
Check e.g. the [PotentialFlowApp](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CompressiblePotentialFlowApplication/CompressiblePotentialFlowApplication.py#L6)

this is needed to avoid nasty errors with undefined symbols

Reply:
done!